packages/module/src/components/CatalogTile/CatalogTile.tsx

```diff
@@ -119,7 +119,7 @@ export class CatalogTile extends React.Component<CatalogTileProps> {
       >
         {(badges.length > 0 || iconImg || iconClass || icon || onClick) && (
           <CardHeader
-            actions={{ actions: badges.length > 0 && this.renderBadges(badges) }}
+            actions={{ actions: badges.length > 0 && this.renderBadges(badges), hasNoOffset: true }}
```

Comment:
The card actions element is positioned above where it would normally flow in the doc since the main use case here is for plain actions. The actions element moves itself up by the value of the top padding of a plain button, so that the top of the icon in that plain button is aligned with the top of the card header. If the actions element just has a normal thing in it (something other than an element with transparent padding on the top of it like a plain button), `hasNoOffset` will position the actions element normally. You may want to adjust that depending on what all goes into the actions element of catalog tiles.